packages/msgraph/src/Utils.js

```diff
@@ -1,13 +1,31 @@
+import { fetch } from 'undici';
 import { composeNextState } from '@openfn/language-common';
 
-export function setUrl(urlParams) {
-  const { apiVersion, resolvePath } = urlParams;
+export function ensureDriveExistsOrFail(drive, fileType) {
```

Comment:
You can just call this `assertDrive`. The word `assert` means "test something or throw", so the functionality would be well represented.

Reply:
Also this should just take `state` and a drive name as parameters. Make it as easy as you can to call it

Reply:
@josephjclark the examples are different, that's why the function takes `drive` and `fileType` because i want to be able to show different errors when `assertDrive()` in `getFolder()` and in `getFile()`